specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/stable/2023-07-20/azurelargeinstance.json

```diff
@@ -0,0 +1,1250 @@
+          "description": "Specifies the network settings for the Azure Large Instance."
+        },
+        "azureLargeInstanceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the Azure Large Instance unique ID."
+        },
+        "powerState": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "starting",
+            "started",
+            "stopping",
+            "stopped",
+            "restarting",
+            "unknown"
+          ],
+          "x-ms-enum": {
+            "name": "AzureLargeInstancePowerStateEnum",
+            "modelAsString": true
+          },
+          "description": "Resource power state"
+        },
+        "proximityPlacementGroup": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource proximity placement group"
+        },
+        "hwRevision": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Hardware revision of an Azure Large Instance"
+        },
+        "partnerNodeId": {
+          "type": "string",
+          "description": "ARM ID of another AzureLargeInstance that will share a network with this AzureLargeInstance"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Accepted",
+            "Creating",
+            "Updating",
+            "Failed",
+            "Succeeded",
+            "Deleting",
+            "Migrating",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "AzureLargeInstanceProvisioningStatesEnum",
+            "modelAsString": true
+          },
+          "description": "State of provisioning of the AzureLargeInstance"
+        }
+      },
+      "description": "Describes the properties of an Azure Large Instance."
+    },
+    "HardwareProfile": {
+      "type": "object",
+      "properties": {
+        "hardwareType": {
+          "readOnly": true,
+          "type": "string",
+          "enum": ["Cisco_UCS", "HPE"],
+          "x-ms-enum": {
+            "name": "AzureLargeInstanceHardwareTypeNamesEnum",
+            "modelAsString": true
+          },
+          "description": "Name of the hardware type (vendor and/or their product name)"
+        },
+        "azureLargeInstanceSize": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "S72m",
+            "S144m",
+            "S72",
+            "S144",
+            "S192",
+            "S192m",
+            "S192xm",
+            "S96",
+            "S112",
+            "S224",
+            "S224m",
+            "S224om",
+            "S224oo",
+            "S224oom",
+            "S224ooo",
+            "S224se",
+            "S448se",
+            "S384",
+            "S384m",
+            "S384xm",
+            "S384xxm",
+            "S448",
```

Comment:
Can we add S448SE that is the one that our new customers are going to use?

Reply:
S448SE was added above on line 922 but I'll move it in order